src/components/slideshow/slideshow.js

```diff
@@ -17,6 +17,9 @@ import ServerConnections from '../ServerConnections';
 import { Swiper } from 'swiper/swiper-bundle.esm';
 import 'swiper/swiper-bundle.css';
 import screenfull from 'screenfull';
+import actionSheet from '../actionSheet/actionSheet';
+import globalize from '../../scripts/globalize';
+import * as userSettings from '../../scripts/settings/userSettings';
```

Comment:
_Not tested_
```suggestion
import { currentSettings as userSettings } from '../../scripts/settings/userSettings';
```
To avoid those backward compatibility exports.

Reply:
Done